tests/postgres_tests/test_dependency_checks.py

```diff
@@ -0,0 +1,177 @@
+from django.core import checks
```

Comment:
Not sure `test_dependency_check` is the most adequate name here. Maybe `test_installed_app_checks`?